src/cogs/esports/slash/scrims.py

```diff
@@ -22,7 +22,15 @@ def __init__(self, bot: Quotient):
         super().__init__()
 
     async def can_use_command(self, interaction: discord.Interaction) -> bool:
-        if not any((interaction.user.guild_permissions.manage_guild, Scrim.is_ignorable(interaction.user))):
+        if interaction.guild is None:
+            await interaction.response.send_message(
+                "Application commands can not be used in Private Messages."
+            )
+            return False
```

Comment:
I don't know will this be effective, but slash commands is operational from DM. 

---

![image](https://user-images.githubusercontent.com/37435729/180297651-70ec2c7e-e216-4cb2-bdd9-b9c26e145784.png)

Reply:
wow, didn't know that but I guess there should be a global thing in the lib to apply a check to all the app_commands just like the `bot_check` for message commands